stdlib/2/types.pyi

```diff
@@ -88,9 +88,8 @@ class UnboundMethodType:
     def __init__(self, func: Callable, obj: object) -> None: ...
     def __call__(self, *args: Any, **kwargs: Any) -> Any: ...
 
-class InstanceType:
-    __doc__ = ...  # type: Optional[str]
```

Comment:
Why were these attributes removed? They do indeed exist on instances of old-style classes.

Reply:
They are now inherited from `object`.

Reply:
Oh interesting, they do indeed exist on `object` in Python 2.